src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Core/Builder/ScaffoldRunnerBuilder.cs

```diff
@@ -72,13 +75,42 @@ private void AddDefaultServices()
 
     private void AddCoreServices()
     {
-        Services.AddLogging();
-        Logging.AddCleanConsoleFormatter();
+        //get some logging properties from environment variables
+        var isVerboseEnabled = Environment.GetEnvironmentVariable(ScaffolderConstants.ENABLE_VERBOSE_LOGGING)?.Equals("true", StringComparison.OrdinalIgnoreCase) == true;
+        var isLogToFileEnabled = Environment.GetEnvironmentVariable(ScaffolderConstants.LOG_TO_FILE)?.Equals("true", StringComparison.OrdinalIgnoreCase) == true; ;
+        // Configure Serilog Logger
+        var loggerConfig = new LoggerConfiguration().WriteTo.Sink(new AnsiConsoleSink());
+        loggerConfig.MinimumLevel.Information();
+        if(isVerboseEnabled)
+        {
+            loggerConfig.MinimumLevel.Verbose();
+        }
+
+        if (isLogToFileEnabled)
+        {
+            var currentDirectory = Directory.GetCurrentDirectory();
+            if (!string.IsNullOrEmpty(currentDirectory))
+            {
+                var filePath = $"dotnet-scaffold-{DateTime.UtcNow:yyyy-MM-dd_HH-mm}.log";
```

Comment:
Agreed that it would be better to have a single folder to view all log files.

Reply:
maybe in the .dotnet-scaffold folder then? `%USERPROFILE%\.dotnet-scaffold\.logs\`?

Reply:
That's probably fine. %TEMP% is often used to make sure it can get cleared out automatically at some point. But since we don't do any automatic logging (its all opt-in) its probably not necessary.

Reply:
logging to %USERPROFILE%\.dotnet-scaffold\.logs\ for now then, fixed.